README.md

```diff
@@ -34,8 +36,31 @@ ident_list = _{ !digit ~ ident ~ (" " ~ ident)+ }
           // ident_list rule is silent which means it produces no tokens
 ```
 
-This is then saved in a `.pest` grammar file and is never mixed up with Rust code which results in an always up-to-date
-formal definition of the grammar which is very easy to maintain.
+Grammars are saved in separate .pest files which are never mixed with procedural code. This results in an always 
+up-to-date formalization of a language that is easy to read and maintain.
+
+## Meaningful error reporting
+
+Based on the grammar definition, the parser also includes automatic error reporting. For the example above, the input 
+"123" will result in: 
+
+```
+thread 'main' panicked at ' --> 1:1
+  |
+1 | 123
+  | ^---
+  |
+  = unexpected digit', src/main.rs:12
+```
+while "ab *" will result in: 
```

Comment:
Probably want to escape this  `*` since GitHub in the source diff thinks everything after should be italicized.

```md
while "ab \*" will result in:
```

while "ab \*" will result in:

Reply:
Alternatively, use a code block here.

```md
while `ab *` will result in:
```

while `ab *` will result in:

```md
while "`ab *`" will result in:
```

while "`ab *`" will result in: